src/pages/homePage/home.jsx

```diff
@@ -0,0 +1,174 @@
+import '../homePage/home.css'
+import { Navbar } from "../../components/navbar/navbar"
+import eBike from '../../assets/images/bike-electric.webp';
+import scooter from '../../assets/images/scooter.png';
+import wishlist from '../../assets/images/wishlist.png';
+import { Footer } from "../../components/footer/footer";
+import { Link } from 'react-router-dom';
+
+
+export const Home = () => {
+    return (
+        <div className="home-container">
+            <Navbar />
+            <main className="main-container">
```

Comment:
Divide this main into smaller components

Reply:
yes, i will divide them in next pr's

Reply:
thank you, i will do the changes.